src/test/parse/parse.inline.test.ts

```diff
@@ -137,4 +137,16 @@ describe("Set Inline", () => {
         result = setInlineField(input, "completion", "2021-02-22");
         expect(result).toEqual("- [x] a completed task [completion:: 2021-02-22] foo bar");
     });
+
+    test("Add emoji shorthand annotation", () => {
+        let input = "- [ ] an uncompleted task";
+        let result = setEmojiShorthandCompletionField(input);
+        expect(result).toEqual(input);
+
+        result = setEmojiShorthandCompletionField(input, "2022-07-26");
```

Comment:
I don't understand the difference between the two `setEmojiShorthandCompletionField()`  calls. It appears that the one on line 143 does nothing, so what behaviour is it testing?

Reply:
I wonder if the call in line 143 is how to remove an existing done-date emoji? But the input to the test does not have a done-date emoji, so I'm not sure that this is what it is testing.

I tested manually that if I completed a task, in a dataview results block, it added a done emoji and date (when this was enabled in my settings)
And I tested that un-completing that same task removed the emoji and date.

Maybe it was in the earlier PR, but I'm not seeing here any test of the removing of the done emoji and date when a task is toggled off again...

Oh wait: this is a testing of parsing, not of clicking... in which case, maybe everything I just said is irrelevant. Apologies if that is the case, but maybe it's a bit confusing that setting of the date is being testing in a file with `parse` in the name.

This may all just come from my unfamiliarity with this code.... Feel free to ignore it!!!